src/utils/GroupCallUtils.ts

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { EventTimeline, MatrixClient, MatrixEvent, RoomState } from "matrix-js-sdk/src/matrix";
+import { deepCopy } from "matrix-js-sdk/src/utils";
+
+export const STUCK_DEVICE_TIMEOUT_MS = 1000 * 60 * 60; // 1 hour
+
+const CALL_STATE_EVENT_TYPE = "org.matrix.msc3401.call";
+const CALL_MEMBER_STATE_EVENT_TYPE = "org.matrix.msc3401.call.member";
+const CALL_STATE_EVENT_TERMINATED = "m.terminated";
```

Comment:
Should we be using unstable thingy from js-sdk here?

Reply:
Not sure about where to put these @robintown ?

Reply:
It's not about where to put them but whether or not to use `UnstableValue`

Reply:
Got your point :+1: There is already a branch providing the event types in the JS SDK. But I am not sure when it will land or if we want to wait for it. This is why I pinged robin here.

Reply:
I thought we would keep them here and use `UnstableValue` - not sure if I've communicated that well...

Reply:
Yes, the JS SDK branch is still going to take a while to land, so `UnstableValue`s here would be the best solution for now

Reply:
I've added two `UnstableValue`s for the event types. Or does the prefix apply to the properties as well?

Reply:
The prefix doesn't apply to the properties, it seems. In Element Call, we're using `m.calls`, `m.devices`, etc. as the property names